src/swaps/boltz.rs

```diff
@@ -310,16 +310,42 @@ impl Fees {
     }
 }
 
+/// States for a submarine swap.
+///
+/// See <https://docs.boltz.exchange/v/api/lifecycle#normal-submarine-swaps>
 #[derive(Serialize, Deserialize, Debug)]
 #[serde(rename_all = "snake_case")]
 pub enum SubSwapStates {
+    /// Initial state of the swap; optionally the initial state can also be `invoice.set` in case
+    /// the invoice was already specified in the `/createswap` request.
```

Comment:
```suggestion
    /// the invoice was already specified in the request that created the swap.
```
It's not `/createswap` in the API v2